<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LexSearchForm</class>
 <widget class="QWidget" name="LexSearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Perform searches that ignore diacritics.
Search term must contain no diacritics.</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
     border: 0px solid black;
     border-radius: 8px;
     padding: 0px;
	 background-color: rgb(255, 255, 255);
}

QLineEdit {
     border: 0px solid black;
     border-radius: 8px;
     padding: 0px;
	 background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QGroupBox" name="lSearchHeader">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>-2</y>
     <width>965</width>
     <height>138</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    padding-left: 3px;
}</string>
   </property>
   <property name="title">
    <string>Lexicographic info</string>
   </property>
   <widget class="QLineEdit" name="lSearchOrth">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>24</y>
      <width>473</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>search orthographic form</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     border: 0px solid black;
     border-radius: 0px;
     padding: 0px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchIPA">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>68</y>
      <width>473</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>search phonemic form</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     border: 0px solid black;
     border-radius: 0px;
     padding: 0px;
}</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="notations">
    <property name="geometry">
     <rect>
      <x>703</x>
      <y>22</y>
      <width>194</width>
      <height>112</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>194</width>
      <height>112</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>194</width>
      <height>112</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lCatSearch">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>7</y>
       <width>50</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>search grammatical labels (e.g., &quot;pl.&quot;)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lGrmSearch">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>7</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>search for grammatical forms</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lC2Search">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>42</y>
       <width>178</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>178</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>178</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>search for comparisons</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lCfSearch">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>77</y>
       <width>178</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>178</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>178</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>search cross-references</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lSearchLit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>104</y>
      <width>633</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search literal glosses</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchPOS">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>28</y>
      <width>57</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search by part of speech</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchRegister">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>70</y>
      <width>57</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search by register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchBrrw">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>69</y>
      <width>140</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search borrowings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchDialectName">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>30</y>
      <width>45</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search by dialects</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lDialectSearch">
    <property name="geometry">
     <rect>
      <x>606</x>
      <y>30</y>
      <width>87</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search for dialectal forms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lGrmDialectSearch">
    <property name="geometry">
     <rect>
      <x>894</x>
      <y>29</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>28</height>
     </size>
    </property>
    <property name="toolTip">
     <string>search for dialect-specific 
grammatical forms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lC2DialectSearch">
    <property name="geometry">
     <rect>
      <x>894</x>
      <y>64</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>28</height>
     </size>
    </property>
    <property name="toolTip">
     <string>search for dialect-specific 
alternative forms</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="lMetadataBox">
   <property name="geometry">
    <rect>
     <x>976</x>
     <y>-2</y>
     <width>148</width>
     <height>138</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 9;
    border-radius: 0pts;
    max-height: 16px;
    min-width: 45px;
    min-height: 16px;
    padding: 1px;
}

QLabel {
	background: transparent;
}</string>
   </property>
   <property name="title">
    <string>Metadata</string>
   </property>
   <widget class="QLabel" name="lLabelSource">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>22</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabelDate">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>59</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabelResearcher">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>22</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Researcher</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabelConfirmed">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>96</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Confirmed</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabelUpdated">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>59</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Updated</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchSource">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>39</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search sources</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchResearcher">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>39</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search researchers</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchDate">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>76</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search by date of entry</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchUpdate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>76</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>serach by date of updates</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lSearchConfirmed">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>113</y>
      <width>132</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search for confirmations by consultant</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="lDefBox">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>136</y>
     <width>732</width>
     <height>338</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Definitions</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="lL1Box">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Primary language</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="lL1Search">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>26</y>
         <width>689</width>
         <height>116</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>search primary language gloss</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="lL2Box">
      <property name="title">
       <string>Secondary language</string>
      </property>
      <widget class="QLineEdit" name="lL2Search">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>26</y>
         <width>689</width>
         <height>116</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>search secondary language gloss</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="lIndexBox">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>472</y>
     <width>731</width>
     <height>112</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Indices</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="lKeywordsBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox.title {
	background-color: transparent;
}</string>
      </property>
      <property name="title">
       <string>Keywords</string>
      </property>
      <widget class="QLineEdit" name="lSearchKeywordIndex">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>25</y>
         <width>201</width>
         <height>43</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>search keywords</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="lPrimaryIndexBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Primary</string>
      </property>
      <widget class="QLineEdit" name="lSearchPrimaryIndex">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>25</y>
         <width>207</width>
         <height>43</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>search primary language index</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="lSecondaryIndexBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Secondary</string>
      </property>
      <widget class="QLineEdit" name="lSearchSecondaryIndex">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>25</y>
         <width>207</width>
         <height>43</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>search secondary language index</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="instructionBox">
   <property name="geometry">
    <rect>
     <x>912</x>
     <y>136</y>
     <width>210</width>
     <height>448</height>
    </rect>
   </property>
   <property name="title">
    <string>Instructions</string>
   </property>
   <widget class="QTextEdit" name="instruction">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>27</y>
      <width>190</width>
      <height>410</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="sDoneBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>117</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="toolTip">
    <string>Check box to search only completed entries.
Clear box to search for unfinished entries.</string>
   </property>
   <property name="text">
    <string>Done?</string>
   </property>
   <property name="tristate">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="lSearchControlBar">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>596</y>
     <width>258</width>
     <height>54</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>258</width>
     <height>54</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>258</width>
     <height>54</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="doSearch">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Perform search</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="lCancelSearch">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Cancel and go back to lexicon</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearForm">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Clear all fields</string>
      </property>
      <property name="text">
       <string>Clear form</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>doSearch</zorder>
   <zorder>clearForm</zorder>
   <zorder>lCancelSearch</zorder>
  </widget>
  <widget class="QGroupBox" name="searchParam">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>136</y>
     <width>147</width>
     <height>213</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {font-size: 10pts; font-style: italic;}</string>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>19</y>
      <width>131</width>
      <height>193</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="caseBtn">
       <property name="toolTip">
        <string>Perform case-insensitive searches.</string>
       </property>
       <property name="text">
        <string> Ignore case</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="accentBtn">
       <property name="toolTip">
        <string>Perform searches that ignore lexical accent.
Search term must contain no accented characters.</string>
       </property>
       <property name="text">
        <string> Ignore accents</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="diacritBtn">
       <property name="text">
        <string> Ignore diacrits</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="appendBtn">
       <property name="toolTip">
        <string>Search results will not overwrite the results
of previous searches on &quot;Search&quot; card.</string>
       </property>
       <property name="text">
        <string> Append results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wholeWordBtn">
       <property name="toolTip">
        <string>Find whole words only.</string>
       </property>
       <property name="text">
        <string> Whole word</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="setLangBtn">
       <property name="toolTip">
        <string>Return results in secondary language.</string>
       </property>
       <property name="text">
        <string> Secondary lang.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lSearchNotes">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>596</y>
     <width>403</width>
     <height>54</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
